README.md

```diff
@@ -167,6 +167,12 @@ This will provide access to API Gateway at [http://localhost:3001](http://localh
 
 Additionally, this ties in with the `serverless webpack` plugin which will ensure that your lambdas are re-built when changes are detected.
 
+### Invoking lambdas locally
+
```

Comment:
Add note for at least some of the lambda it seems we have to set `export IS_OFFLINE=true`